src/main/java/com/commercetools/sync/sdk2/commons/utils/CompletableFutureUtils.java

```diff
@@ -0,0 +1,178 @@
+package com.commercetools.sync.sdk2.commons.utils;
+
+import static com.commercetools.sync.sdk2.commons.utils.StreamUtils.filterNullAndMap;
+import static java.util.stream.Collectors.toList;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Function;
+import java.util.stream.Collector;
+import java.util.stream.Stream;
+import javax.annotation.Nonnull;
+
+public final class CompletableFutureUtils {
```

Comment:
This class was just added in v14.0.0 of java sdk v2: https://github.com/commercetools/commercetools-sdk-java-v2/pull/430

Please remove this class and use the one from the sdk.

Reply:
https://commercetools.atlassian.net/browse/DEVX-58